pom.xml

```diff
@@ -139,13 +139,13 @@
         <geronimo-jms_2_spec.version>1.0-alpha-2</geronimo-jms_2_spec.version>
         <!--END versions that MUST exactly match the versions used by the CXF and Camel features-->
         <!-- Artemis dependencies -->
-        <artemis.version>1.5.1</artemis.version>
-        <proton.version>0.15.0</proton.version>
+        <artemis.version>2.1.0</artemis.version>
+        <proton.version>0.19.0</proton.version>
         <jboss-logging.version>3.3.0.Final</jboss-logging.version>
-        <jgroups.version>3.6.9.Final</jgroups.version>
+        <jgroups.version>3.6.13.Final</jgroups.version>
         <commons-beanutils.version>1.9.2</commons-beanutils.version>
-        <netty.version>4.1.5.Final</netty.version>
-        <qpid-jms.version>0.11.0</qpid-jms.version>
+        <netty.version>4.1.9.Final</netty.version>
+        <qpid-jms.version>0.23.0</qpid-jms.version>
```

Comment:
Could use a note here that camel must use the same version that artemis uses so that it doesn't get missed in the next camel/artemis upgrade